pkgs/tools/misc/nginx-config-formatter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "2016-06-16";
+  name = "nginx-config-formatter-${version}";
```

Comment:
I'm of the opinion that if there isn't any stable release that we just drop the `unstable`. As soon as there will be a release we would likely use that instead of this version. In that case the name remains the same. If you would now append `-unstable` to the name part, the name changes and it would break `nix-env -u`.